src/rollup/types.d.ts

```diff
@@ -446,11 +447,10 @@ export interface RenderedModule {
 	renderedLength: number;
 }
 
-export interface RenderedChunk {
+export interface PreRenderedChunk {
```

Comment:
I'm not sure about this but passing the fileName to augmentChunkHash hook seemed wrong since its not yet computed so I created a reduced interface with all the other info